### PR TITLE
Introduce the usage of `LoadBalancer` typed Kubernetes services

### PR DESCRIPTION
**Pull request by chirangaalwis**

### CODE REVIEW

File: scalable-integrator/integrator-deployment.yaml [hunk 1]
```diff
@@ -82,4 +82,4 @@ spec:
           name: integrator-conf-datasources
       - name: shared-pd
         persistentVolumeClaim:
-          claimName: shared-volume-claim
+          claimName: integrator-server-share-volume-claim
```
Comment (DilanUA) [inline scalable-integrator/integrator-deployment.yaml]: Shall we name this as ""scalable-integrator-shared-volume-claim" ?

File: scalable-integrator/volumes/persistent-volumes.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@
 apiVersion: v1
 kind: PersistentVolume
 metadata:
-  name: shared-persistent-volume
+  name: integrator-server-share-persistent-volume
```
Comment (DilanUA) [inline scalable-integrator/volumes/persistent-volumes.yaml]: Shall we rename this to "scalable-integrator-shared-persistent-volume" ?

File: scalable-integrator/integrator-deployment.yaml (hunk 1 above)
Comment (chirangaalwis) [reply scalable-integrator/integrator-deployment.yaml]: +1

File: scalable-integrator/volumes/persistent-volumes.yaml (hunk 2 above)
Comment (chirangaalwis) [reply scalable-integrator/volumes/persistent-volumes.yaml]: +1

**Pull request merged**